libs/tup-components/src/tickets/TicketDetailModal/TicketReplyForm.tsx

```diff
@@ -55,11 +72,27 @@ export const TicketReplyForm: React.FC<{ ticketId: string }> = ({
   return (
     <Formik
       enableReinitialize
+      // enable this line 25 for form validation.
+      // validationSchema={formSchema}
```

Comment:
Why has validation been disabled here?

Reply:
The formSchema that's created on Line 19 does not allow for customizable validation (at least in a way that I know how). We want to get rid of the required-ness of the reply textarea when the Ticket Status checkbox is checked, and bring it back when it is not.

Reply:
OK. Formik should support custom validators, but since this is such a simple form I think it's acceptable to rely on standard HTML behavior.

Reply:
Good idea. I would have looked into that had I thought of it. The simpleness of the form could make it a good minimal example of customization. Ticket created and backlogged:

- [TUP-684](https://tacc-main.atlassian.net/browse/TUP-684): In Ticket Modal Try Custom Formik Validation